server/routes/services.js

```diff
@@ -0,0 +1,36 @@
+class WeatherApi {
+    constructor() {
+        this.apiKey =process.env.API_KEY
+        this.photoApiKey = process.env.PHOTOS_API_KEY
+        this.axios = require('axios')
+    }
+    async getWetherBycity(cityName) {
+        return  this.axios.get(`https://api.openweathermap.org/data/2.5/weather?q=${cityName}&appid=${this.apiKey}&units=metric`)
```

Comment:
maybe also declare the basic url in the constructor as well to make it cleaner
`this.weatherAPI = https://api.openweathermap.org/data/2.5/`